docs/installation.md

```diff
@@ -51,16 +73,27 @@ $ npm init --yes
 
 2. install required tools
 ```bash
-$ npm install mithril --save
-$ npm install webpack webpack-cli --save-dev
+$ npm install mithril
+$ npm install esbuild --save-dev
 ```
 
 3. Add a "start" entry to the scripts section in `package.json`.
 ```json
 {
 	"...": "...",
 	"scripts": {
-		"start": "webpack ./src/index.js --output-path ./bin --watch"
+		"start": "esbuild ./src/index.js --bundle --outfile=./bin/main.js --watch"
+	}
+}
+```
+
+Optionally, if you'd like to use JSX, you can use the `--jsx-factory` and `--jsx-fragment` flags with esbuild.
```

Comment:
The indentation looks not so nice. This could maybe optimized?

<img width="515" alt="indentation" src="https://github.com/MithrilJS/mithril.js/assets/1334161/4173cfa0-8cf5-4d1e-b7a1-5bda4d5c4319">

Reply:
Fixed in e51feac36cd14feba1fdff46e84720c7bb9bc9e3